boa_restrictor/common/rule.py

```diff
@@ -0,0 +1,25 @@
+from boa_restrictor.projections.occurrence import Occurrence
+
+LINTING_RULE_PREFIX = "PBR"
+
+
+class Rule:
+    RULE_ID: str
+    RULE_LABEL: str
+
+    filename: str
+    source_code: str
+
+    @classmethod
+    def run_check(cls, *, filename: str, source_code: str) -> list[Occurrence]:
+        instance = cls(filename=filename, source_code=source_code)
+        return instance.check(source_code=source_code)
+
+    def __init__(self, *, filename: str, source_code: str):
+        super().__init__()
+
+        self.filename = filename
```

Comment:
question: Interessiert der Dateiname fürs Linting?

Reply:
Ich brauch den in der Ausgabe, darum muss ich den mitnehmen.